Assets/HoloToolkit/SpatialMapping/Scripts/SpatialProcessing/RoomSaver.cs

```diff
@@ -0,0 +1,116 @@
+using HoloToolkit.Unity;
+using UnityEngine;
+using UnityEngine.VR.WSA;
+using UnityEngine.VR.WSA.Persistence;
+using System.Collections.Generic;
+using HoloToolkit.Unity.SpatialMapping;
+
+public class RoomSaver : MonoBehaviour
+{
+
+    public string fileName;             // name of file to store meshes
+    public string anchorStoreName;      // name of world anchor to store for room
+
+    List<MeshFilter> roomMeshFilters;
+    WorldAnchorStore anchorStore;
+    int meshCount = 0;
+
+    // Use this for initialization
+    void Start()
+    {
+        WorldAnchorStore.GetAsync(AnchorStoreReady);
+    }
+
+    void AnchorStoreReady(WorldAnchorStore store)
+    {
+        anchorStore = store;
+    }
+
+    public bool SaveRoom()
+    {
+        // if the anchor store is not ready then we cannot save the room mesh
+        if (anchorStore == null)
+        {
+            return false;
+        }
+
+        // delete old relevant anchors
+        string[] anchorIds = anchorStore.GetAllIds();
+        for (int i = 0; i < anchorIds.Length; i++)
+        {
+            if (anchorIds[i].Contains(anchorStoreName))
+            {
+                anchorStore.Delete(anchorIds[i]);
+            }
+        }
+
+        Debug.Log("Old anchors deleted...");
```

Comment:
Do we really need all these debug logs?
Comments should be enough, if you're no longer testing